oauthdoctor/diag/config.go

```diff
@@ -452,9 +452,14 @@ func GetDefaultConfigFile(lang string) (ConfigFile, error) {
 	return cfg, nil
 }
 
+// printout prints out key and value in the log. This is used for unit testing.
```

Comment:
Remove the comment that this is used for unit testing. We may use it for that, but we should not say that it is restricted to that. I'm not clear what value it adds.